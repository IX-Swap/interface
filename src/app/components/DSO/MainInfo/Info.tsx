import { Grid, Typography } from '@mui/material'
import React from 'react'

export interface InfoProps {
  label: string
  value?: string | number
}
export const Info = ({ label, value }: InfoProps) => {
  return (
    <Grid container spacing={0} justifyContent='space-between'>
      <Grid itemScope>
        <Typography variant='subtitle1' color='textSecondary'>
          {label}
        </Typography>
      </Grid>
      <Grid item>
        <Typography variant='subtitle1' textAlign='right'>
          {value !== undefined ? value : '-'}
        </Typography>
      </Grid>
    </Grid>
  )
}
