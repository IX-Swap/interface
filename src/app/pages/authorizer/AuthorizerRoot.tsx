import React from 'react'
import { AuthorizerRouter } from 'app/pages/authorizer/router/AuthorizerRouter'

export const AuthorizerRoot: React.FC = () => {
  return <AuthorizerRouter />
}
