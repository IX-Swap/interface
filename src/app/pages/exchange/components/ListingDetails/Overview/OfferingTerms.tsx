import { Grid } from '@material-ui/core'
import { FormSectionHeader } from 'app/components/DSO/components/FormSectionHeader'
import { PercentageNumber } from 'app/components/DSO/DSOPreview/PercentageNumber'
import { LabelledValue } from 'components/LabelledValue'
import React from 'react'

export interface OfferingTermsProps {
  investmentPeriod: number
  dividendYield: number
  investmentStructure: string
  grossIrr: number
  equityMultiple: number
  distributionFrequency: string
  leverage: number
  interestRate: number
  capitalStructure: string
}

export const OfferingTerms = ({
  investmentPeriod,
  dividendYield,
  investmentStructure,
  grossIrr,
  equityMultiple,
  distributionFrequency,
  leverage,
  interestRate,
  capitalStructure
}: OfferingTermsProps) => {
  const isDebt = capitalStructure === 'Debt'
  const isEquity = capitalStructure === 'Equity'

  return (
    <Grid container spacing={2}>
      <Grid item xs={12}>
        <FormSectionHeader title='Offering Terms' />
      </Grid>

      <Grid item xs={12} sm={6} md={4}>
        <LabelledValue
          label='Investment Period'
          value={`${investmentPeriod} months`}
        />
      </Grid>

      {!isDebt ? (
        <>
          <Grid item xs={12} sm={6} md={4}>
            <LabelledValue
              label='Dividend Yield'
              value={<PercentageNumber value={dividendYield} />}
            />
          </Grid>
          <Grid item xs={12} sm={6} md={4}>
            <LabelledValue
              label='Gross IRR (%)'
              value={<PercentageNumber value={grossIrr} />}
            />
          </Grid>
          <Grid item xs={12} sm={6} md={4}>
            <LabelledValue
              label='Equity Multiple'
              value={<PercentageNumber value={equityMultiple} />}
            />
          </Grid>
        </>
      ) : null}

      {!isEquity ? (
        <>
          <Grid item xs={12} sm={6} md={4}>
            <LabelledValue
              label='Interest Rate'
              value={<PercentageNumber value={interestRate} />}
            />
          </Grid>
          <Grid item xs={12} sm={6} md={4}>
            <LabelledValue
              label='Leverage'
              value={<PercentageNumber value={leverage} />}
            />
          </Grid>
        </>
      ) : null}

      <Grid item xs={12} sm={6} md={4}>
        <LabelledValue
          label='Investment Structure'
          value={investmentStructure}
        />
      </Grid>

      <Grid item xs={12} sm={6} md={4}>
        <LabelledValue
          label='Distribution Frequency'
          value={distributionFrequency}
        />
      </Grid>
    </Grid>
  )
}
