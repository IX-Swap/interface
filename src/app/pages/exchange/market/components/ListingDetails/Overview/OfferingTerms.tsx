import { Grid } from '@material-ui/core'
import { FormSectionHeader } from 'app/components/DSO/components/FormSectionHeader'
import { LabelledValue } from 'components/LabelledValue'
import React from 'react'

export const OfferingTerms = () => {
  return (
    <Grid container spacing={2}>
      <Grid item xs={12}>
        <FormSectionHeader title='Offering Terms' />
      </Grid>
      <Grid item xs={12} sm={6} md={4}>
        <LabelledValue label='Investment Period' value='12' />
      </Grid>
      <Grid item xs={12} sm={6} md={8}>
        <LabelledValue label='Dividend Yield' value='12' />
      </Grid>
      <Grid item xs={12} sm={6} md={4}>
        <LabelledValue label='Investment Structure' value='Equity' />
      </Grid>
      <Grid item xs={12} sm={6} md={8}>
        <LabelledValue label='Gross IRR (%)' value='5' />
      </Grid>
      <Grid item xs={12} sm={6} md={4}>
        <LabelledValue label='Equity Multiple' value='2' />
      </Grid>
      <Grid item xs={12} sm={6} md={8}>
        <LabelledValue label='Distribution Frequency' value='Quaterly' />
      </Grid>
    </Grid>
  )
}
