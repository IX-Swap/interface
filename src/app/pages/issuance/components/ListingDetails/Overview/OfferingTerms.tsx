import { PercentageNumber } from 'app/components/DSO/DSOPreview/PercentageNumber'
import React from 'react'
import { FieldGrid } from 'ui/FieldGrid/FieldGrid'

export interface OfferingTermsProps {
  investmentPeriod: number
  dividendYield: number
  investmentStructure: string
  grossIrr: number
  equityMultiple: number
  distributionFrequency: string
  leverage: number
  interestRate: number
  capitalStructure: string
}

export const OfferingTerms = ({
  investmentPeriod,
  dividendYield,
  investmentStructure,
  grossIrr,
  equityMultiple,
  distributionFrequency,
  leverage,
  interestRate,
  capitalStructure
}: OfferingTermsProps) => {
  const isDebt = capitalStructure === 'Debt'
  const isEquity = capitalStructure === 'Equity'

  const items = [
    {
      label: 'Investment Period',
      value: `${investmentPeriod} months`
    },

    !isDebt && {
      label: 'Dividend Yield (%)',
      value: <PercentageNumber value={dividendYield} />
    },
    !isDebt && {
      label: 'Gross IRR (%)',
      value: <PercentageNumber value={grossIrr} />
    },
    !isDebt && {
      label: 'Equity Multiple',
      value: <PercentageNumber value={equityMultiple} />
    },
    !isEquity && {
      label: 'Interest Rate',
      value: <PercentageNumber value={interestRate} />
    },
    !isEquity && {
      label: 'Leverage',
      value: <PercentageNumber value={leverage} />
    },
    {
      label: 'Investment Structure',
      value: investmentStructure
    },
    {
      label: 'Distribution Frequency',
      value: distributionFrequency
    }
  ]

  return <FieldGrid title={'Offering Terms'} items={items.filter(Boolean)} />
}
