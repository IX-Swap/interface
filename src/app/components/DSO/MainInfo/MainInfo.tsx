import { Grid, Paper, Typography } from '@mui/material'
import { Info } from 'app/components/DSO/MainInfo/Info'
import { Expandable } from 'app/components/Expandable/Expandable'
import { formatDateToMMDDYY } from 'helpers/dates'
import { useAppBreakpoints } from 'hooks/useAppBreakpoints'
import React from 'react'
import { DigitalSecurityOffering } from 'types/dso'

export interface MainInfoProps {
  dso: DigitalSecurityOffering
}

export const MainInfo = ({ dso }: MainInfoProps) => {
  const { isTablet } = useAppBreakpoints()

  const expandedComponent = () => (
    <>
      <Grid item xs={12}>
        <Info label='Network' value={dso.network?.name} />
      </Grid>
      <Grid item xs={12}>
        <Info label='Decimal' value={dso.decimals} />
      </Grid>
      <Grid item xs={12}>
        <Info label='Capital Structure' value={dso.capitalStructure} />
      </Grid>
      <Grid item xs={12}>
        <Info label='Launch Date' value={formatDateToMMDDYY(dso.launchDate)} />
      </Grid>
      <Grid item xs={12}>
        <Info
          label='Completion Date'
          value={formatDateToMMDDYY(dso.completionDate)}
        />
      </Grid>
    </>
  )

  return (
    <Paper sx={{ p: { xs: 3, md: 5 }, borderRadius: 2 }}>
      {isTablet ? (
        <Expandable
          noBorders
          spacing={3}
          showArrow
          mainComponent={<Typography variant='h4'>Main Info</Typography>}
          expandedComponent={
            <Grid container spacing={3}>
              {expandedComponent()}
            </Grid>
          }
        />
      ) : (
        <Grid container spacing={3}>
          <Grid item xs={12}>
            <Typography variant='h4'>Main Info</Typography>
          </Grid>
          {expandedComponent()}
        </Grid>
      )}
    </Paper>
  )
}
