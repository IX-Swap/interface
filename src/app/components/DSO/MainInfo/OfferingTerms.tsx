import { Grid, Paper, Typography } from '@mui/material'
import { Info } from 'app/components/DSO/MainInfo/Info'
import { Expandable } from 'app/components/Expandable/Expandable'
import { getPercentageValue } from 'config/utils'
import { useAppBreakpoints } from 'hooks/useAppBreakpoints'
import React from 'react'
import { DigitalSecurityOffering } from 'types/dso'

export interface OfferingTermsProps {
  dso: DigitalSecurityOffering
}

export const OfferingTerms = ({ dso }: OfferingTermsProps) => {
  const { isTablet } = useAppBreakpoints()
  const dividendYield =
    dso.dividendYield !== undefined
      ? `${getPercentageValue(dso.dividendYield)}%`
      : undefined
  const interestRate =
    dso.interestRate !== undefined
      ? `${getPercentageValue(dso.interestRate)}%`
      : undefined
  const grossIRR =
    dso.grossIRR !== undefined
      ? `${getPercentageValue(dso.grossIRR)}%`
      : undefined

  const equityMultiple =
    dso.equityMultiple !== undefined
      ? `${getPercentageValue(dso.equityMultiple)}%`
      : undefined

  const investmentPeriod =
    dso.investmentPeriod !== undefined
      ? `${dso.investmentPeriod} ${
          dso.investmentPeriod === 1 ? 'Month' : 'Months'
        }`
      : undefined

  const expandedComponent = () => (
    <>
      <Grid item xs={12}>
        <Info label='Investment Period' value={investmentPeriod} />
      </Grid>
      <Grid item xs={12}>
        <Info label='Dividend Yield' value={dividendYield} />
      </Grid>
      <Grid item xs={12}>
        <Info label='Interest Rate' value={interestRate} />
      </Grid>
      <Grid item xs={12}>
        <Info label='Investment Structure' value={dso.investmentStructure} />
      </Grid>
      <Grid item xs={12}>
        <Info label='Gross IRR' value={grossIRR} />
      </Grid>
      <Grid item xs={12}>
        <Info label='Leverage' value={dso.leverage} />
      </Grid>
      <Grid item xs={12}>
        <Info label='Equity Multiple' value={equityMultiple} />
      </Grid>
      <Grid item xs={12}>
        <Info
          label='Distribution Frequency'
          value={dso.distributionFrequency}
        />
      </Grid>
    </>
  )
  return (
    <Paper sx={{ p: { xs: 3, md: 5 }, borderRadius: 2 }}>
      {isTablet ? (
        <Expandable
          noBorders
          spacing={3}
          showArrow
          mainComponent={<Typography variant='h4'>Offering Terms</Typography>}
          expandedComponent={
            <Grid container spacing={3}>
              {expandedComponent()}
            </Grid>
          }
        />
      ) : (
        <Grid container spacing={3}>
          <Grid item xs={12}>
            <Typography variant='h4'>Offering Terms</Typography>
          </Grid>
          {expandedComponent()}
        </Grid>
      )}
    </Paper>
  )
}
