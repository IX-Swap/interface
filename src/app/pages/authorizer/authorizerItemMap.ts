import { AppFeature, AuthorizerCategory } from 'types/app'
import { BankPreview } from 'app/components/BankPreview/BankPreview'
import { urlParams } from 'config/appURL'
import { DSWithdrawalPreview } from 'app/components/DSWithdrawalPreview/DSWithdrawalPreview'
import { WithdrawalPreview } from 'app/components/WithdrawalPreview/WithdrawalPreview'
import { DepositView } from 'app/components/DepositView/DepositView'
import { WithdrawalAddressPreview } from 'app/components/WithdrawalAddressPreview/WithdrawalAddressPreview'
import { CommitmentPreview } from 'app/components/CommitmentPreview/CommitmentPreview'
import { DSOView } from 'app/components/DSO/DSOView'
import { CorporateView } from 'app/pages/identity/components/CorporateView'
import { IndividualView } from 'app/pages/identity/components/IndividualView'
import { AuthorizerRoute } from 'app/pages/authorizer/router'

export interface AuthorizerMapItem {
  uri: string
  component: any
  paramKey: string
  title: string
  listRoute: keyof typeof AuthorizerRoute
}

export const authorizerItemMap: Record<
  AuthorizerCategory,
  AuthorizerMapItem
> = {
  [AppFeature.BankAccounts]: {
    uri: '/accounts/banks/list',
    component: BankPreview,
    paramKey: urlParams.itemId,
    title: 'Bank Account',
    listRoute: 'banks' as const
  },
  [AppFeature.Offerings]: {
    uri: '/issuance/dso/list',
    component: DSOView,
    paramKey: urlParams.itemId,
    title: 'Digital Security Offering',
    listRoute: 'offerings' as const
  },
  [AppFeature.DigitalSecurityWithdrawals]: {
    uri: '/accounts/security/withdrawals',
    component: DSWithdrawalPreview,
    paramKey: urlParams.itemId,
    title: 'Digital Security Withdrawal',
    listRoute: 'dsWithdrawals' as const
  },
  [AppFeature.CashWithdrawals]: {
    uri: '/accounts/cash/withdrawals',
    component: WithdrawalPreview,
    paramKey: urlParams.itemId,
    title: 'Cash Withdrawal',
    listRoute: 'cashWithdrawals' as const
  },
  [AppFeature.Corporates]: {
    uri: '/identity/corporates/list',
    component: CorporateView,
    paramKey: urlParams.itemId,
    title: 'Corporate Identity',
    listRoute: 'corporateIdentities' as const
  },
  [AppFeature.Individuals]: {
    uri: '/identity/individuals/list',
    component: IndividualView,
    paramKey: urlParams.itemId,
    title: 'Individual Identity',
    listRoute: 'individualIdentities' as const
  },
  [AppFeature.CashDeposits]: {
    uri: '/accounts/cash/deposits',
    component: DepositView,
    paramKey: urlParams.itemId,
    title: 'Cash Deposit',
    listRoute: 'cashDeposits' as const
  },
  [AppFeature.Commitments]: {
    uri: '/issuance/commitments/list',
    component: CommitmentPreview,
    paramKey: urlParams.itemId,
    title: 'Commitment',
    listRoute: 'commitments' as const
  },
  [AppFeature.WithdrawalAddresses]: {
    uri: '/accounts/withdrawal-addresses/list',
    component: WithdrawalAddressPreview,
    paramKey: urlParams.itemId,
    title: 'Withdrawal Address',
    listRoute: 'withdrawalAddresses' as const
  }
}
