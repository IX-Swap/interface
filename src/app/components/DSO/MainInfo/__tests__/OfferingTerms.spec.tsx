import { OfferingTerms } from 'app/components/DSO/MainInfo/OfferingTerms'
import React from 'react'
import { render, cleanup } from 'test-utils'
import { dso } from '__fixtures__/authorizer'

describe('OfferingsTerms', () => {
  afterEach(async () => {
    await cleanup()
    jest.clearAllMocks()
  })

  it('renders without errors', () => {
    const { container } = render(<OfferingTerms dso={dso} />)
    expect(container).toMatchSnapshot()
  })
})
