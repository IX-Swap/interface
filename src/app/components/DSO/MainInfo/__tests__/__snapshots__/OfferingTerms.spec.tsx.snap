// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferingsTerms renders without errors 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation0 css-7ncgri-MuiPaper-root"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3 css-zow5z4-MuiGrid-root"
    >
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
      >
        <h4
          class="MuiTypography-root MuiTypography-h4 css-vxrdkt-MuiTypography-root"
        >
          Offering Terms
        </h4>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
        >
          <div
            class="MuiGrid-root css-vj1n65-MuiGrid-root"
            itemscope=""
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
            >
              Investment Period
            </h6>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
            >
              1
            </h6>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
        >
          <div
            class="MuiGrid-root css-vj1n65-MuiGrid-root"
            itemscope=""
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
            >
              Dividend Yield
            </h6>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
            >
              100%
            </h6>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
        >
          <div
            class="MuiGrid-root css-vj1n65-MuiGrid-root"
            itemscope=""
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
            >
              Interest Rate
            </h6>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
            >
              100%
            </h6>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
        >
          <div
            class="MuiGrid-root css-vj1n65-MuiGrid-root"
            itemscope=""
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
            >
              Investment Structure
            </h6>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
            >
              investment structure
            </h6>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
        >
          <div
            class="MuiGrid-root css-vj1n65-MuiGrid-root"
            itemscope=""
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
            >
              Gross IRR
            </h6>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
            >
              0%
            </h6>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
        >
          <div
            class="MuiGrid-root css-vj1n65-MuiGrid-root"
            itemscope=""
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
            >
              Leverage
            </h6>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
            >
              0
            </h6>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
        >
          <div
            class="MuiGrid-root css-vj1n65-MuiGrid-root"
            itemscope=""
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
            >
              Equity Multiple
            </h6>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
            >
              0%
            </h6>
          </div>
        </div>
      </div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-ewf807-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-1xzqqmo-MuiGrid-root"
        >
          <div
            class="MuiGrid-root css-vj1n65-MuiGrid-root"
            itemscope=""
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-3phns2-MuiTypography-root"
            >
              Distribution Frequency
            </h6>
          </div>
          <div
            class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
          >
            <h6
              class="MuiTypography-root MuiTypography-subtitle1 css-1fjywxu-MuiTypography-root"
            >
              distribution frequency
            </h6>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
